sdk/eventhub/azure-eventhub/azure/eventhub/aio/_client_base_async.py

```diff
@@ -515,6 +515,9 @@ async def _do_retryable_operation(
                 raise
             except Exception as exception:  # pylint:disable=broad-except
                 last_exception = await self._handle_exception(exception)
+                # If optional dependency is not installed, do not retry.
+                if isinstance(exception.__cause__, ModuleNotFoundError):
```

Comment:
In the transport we're still have `ImportError` as the cause - do we need to handle that too?

Reply:
yes updated there!